### DISCUSSION TITLE
Current State of Development?

### DISCUSSION THREAD

**Discussion by DansDesigns** [vd-rd/sbc_allwinner_a13#2] [Ideas]
Hey,
Gotta say, this is brilliant and exactly what I'm looking for!

I have been using a Nano Pi Neo (Allwinner-H3) as the basis for a multi-sensor scanner but I'm having issues with a few constraints imposed by the H3.. 

I'd love to switch over to this board and embed it in my projects!

What is the current state of software for this board?
Have you successfully booted Linux on it? - If so are you using Armbian?

**Comment by vd-rd** [comment]
Thank you very much for the kind feedback.

The current state is hardware validation. Board is back from fab, pending assembly and testing.

Once that is done, u-boot and kernel bringing will commence, since this is a custom board and dts and boards scripts are needed for proper bringing.

So, in a nutshell, i would love to suggest you to use it in your projects, but it's not there yet.
